.gitignore

```diff
@@ -48,7 +48,6 @@ coverage.xml
 .pytest_cache/
 test-results.xml
 test-results/
-__snapshots__/
```

Comment:
not sure this was added to the excusion in this https://github.com/timotheeguerin/typespec/commit/f5ceafda4fc8ba6f2eadbcb27c7767c2a4c4560a#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R51